### ISSUE TITLE
Forcing the PI to do Https breaks the connection

### ISSUE THREAD

**Issue by str3den** [UnchartedBull/OctoDash#2391]
**Describe the bug**
Hi im doing selfsigned signature in octopi and it't breaks the connection to octodash, I tryed editing the config file but it seams there is no support for https??

**Comment by UnchartedBull** [comment]
I don't see an issue with using https. Have you tried editing the config manually?

**Comment by str3den** [comment]
Yes edited the config file manually with Https:// but could not complet the opstart guide.

And once I removed the forced Https on the PI everything worked fine.

Sendt fra Mail<https://go.microsoft.com/fwlink/?LinkId=550986> til Windows

Fra: Timon ***@***.***>
Sendt: 3. november 2021 10:08
Til: ***@***.***>
Cc: ***@***.***>; ***@***.***>
Emne: Re: [UnchartedBull/OctoDash] Forcing the PI to do Https breaks the connection (Issue #2391)


I don't see an issue with using https. Have you tried editing the config manually?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FUnchartedBull%2FOctoDash%2Fissues%2F2391%23issuecomment-958763362&data=04%7C01%7C%7Cebf1edc687b9459d9b8e08d99ea97c84%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637715273043736476%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=%2BMp78uaIkTuNsn2lBI9DuLDDg5M1%2FDYjh%2BIEompyje8%3D&reserved=0>, or unsubscribe<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAWJF3UZQZNANKTX5NKN4ORDUKD3YLANCNFSM5HCKA4KQ&data=04%7C01%7C%7Cebf1edc687b9459d9b8e08d99ea97c84%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637715273043736476%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=iCsRtqo3ZjxariboPZhJht6f03FwYK96%2BAPy5ZTIYX0%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7C%7Cebf1edc687b9459d9b8e08d99ea97c84%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637715273043746473%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=vJWU5JEGWBnJxI3%2F0kbkL5DQM2uTyUOvzyWtTjA4e9A%3D&reserved=0> or Android<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26referrer%3Dutm_campaign%253Dnotification-email%2526utm_medium%253Demail%2526utm_source%253Dgithub&data=04%7C01%7C%7Cebf1edc687b9459d9b8e08d99ea97c84%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637715273043746473%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=JNnCZa9OtRr0w8c1ZKRscF51i0Vmd2Bu7P7krn9YGLE%3D&reserved=0>.

**Comment by UnchartedBull** [comment]
Yes you probably need to create the config completely manually, since the wizard is not setup to accept `https`. So you only can change to `https` after the installation wizard or create the config manually from the ground up.

**Comment by str3den** [comment]
Yea sorry if that wasn't clear, also tryed after the guide completed edit the config file to https the program simply wont accept a adresse with https://

**Comment by UnchartedBull** [comment]
Oh that is interesting. Going to test tomorrow and see how to fix that.
